tools/fw-update/rs-fw-update.cpp

```diff
@@ -119,6 +119,40 @@ void list_devices(rs2::context ctx)
     }
 }
 
+int write_to_mipi_device( const std::vector< uint8_t > & fw_image)
```

Comment:
Can you change the method's name to write_fw_to_mipi_device?

Reply:
It is part of the FW update tool so seemed obvious, and also the parameter states the meaning of the data, but I will update